utils_test.go

```diff
@@ -67,6 +66,29 @@ func AssertResponse(t *testing.T, rr *httptest.ResponseRecorder, code int, want
 	assert.Equal(t, reformatHTML(t, want), reformatHTML(t, rr.Body.String()))
 }
 
+// getTestTemplates returns a [template.Template] object with the default templates,
+// overwritten by the  [overrideTemplates]. If [overrideTemplates] is nil, the returned
+// templates are a clone of the global [_templates].
+func getTestTemplates(t *testing.T, overrideTemplates map[string]string) *template.Template {
```

Comment:
Yeah, I think this can just use `testing.TB`.
```suggestion
func getTestTemplates(t testing.TB, overrideTemplates map[string]string) *template.Template {
```